install/share/ipa-httpd.conf.template

```diff
@@ -1,7 +1,7 @@
 # Do not edit. Created by IPA installer.
 
 [Service]
+Environment=KRB5CCNAME=/tmp/krb5-httpd
```

Comment:
I would rather revive  `paths.KRB5CC_HTTPD` and substitute it to the template than having magic strings.

I also noticed that `KRB5CC_HTTPD` was left over in ipaplatform.debian.paths module.